addition of subdomain note (#8938)
DataDog/documentation: content/en/account_management/saml/okta.md

* **sn**: user.lastName
* **givenName**: user.firstName

> Additional Information on configuring SAML for your Datadog account can be found [on the dedicated SAML documentation page][2]. If you are using the custom sub-domain feature, your specific details will also be found here.

Reviewer: ```suggestion
Additional information on configuring SAML for your Datadog account is available on the [SAML documentation page][2]. If you are using the custom sub-domain feature, your specific details are also available there.
```
